spec/support/integration_test_actions.rb

```diff
@@ -112,4 +115,8 @@ def trim_image_edges(path)
     system!(cmd)
     system!(cmd2)
   end
+
+  def check_custom_enabled
+    pending 'Enable custom repositories from site.defaults.yml' unless SiteSetting.value('enable_custom_repositories')
```

Comment:
Not really needed, but otherwise the tests will simply fail if the setting is disabled. I can remove it if you want.